.gitignore

```diff
@@ -1,6 +1,12 @@
+#VS code
+*.project
+*.settings/
+*.vscode/
+
 *.idea
 
 .gradle
+/build/
 
 # Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
-!gradle-wrapper.jar
\ No newline at end of file
+!gradle-wrapper.jar
```

Comment:
✏️ Are any of these changes in .gitignore related to your PR?

Reply:
Yes, added in some ignores for vs code files and for the gradle build dirs